src/pybind/mgr/test_orchestrator/module.py

```diff
@@ -20,41 +20,7 @@
 from mgr_module import MgrModule
 
 import orchestrator
-
-
-class TestCompletion(orchestrator.Completion):
-    def evaluate(self):
-        self.finalize(None)
-
-
-def deferred_read(f):
-    # type: (Callable) -> Callable[..., TestCompletion]
-    """
-    Decorator to make methods return
-    a completion object that executes themselves.
-    """
-
-    @functools.wraps(f)
-    def wrapper(*args, **kwargs):
-        return TestCompletion(on_complete=lambda _: f(*args, **kwargs))
-
-    return wrapper
-
-
-def deferred_write(message):
-    def inner(f):
-        # type: (Callable) -> Callable[..., TestCompletion]
-
-        @functools.wraps(f)
-        def wrapper(self, *args, **kwargs):
-            return TestCompletion.with_progress(
-                message=message,
-                mgr=self,
-                on_complete=lambda _: f(self, *args, **kwargs),
-            )
-
-        return wrapper
-    return inner
+from orchestrator import handle_orch_error, raise_if_exception
```

Comment:
Instead update the import here

Reply:
I'd like to avoid introducing unrelated changes

Reply:
As you are importing from 'orchestrator' module here again, I suggested the change. If you don't plan do it then at least add a note to clean up later.